packages/medusa/src/api/routes/store/orders/index.ts

```diff
@@ -83,6 +107,23 @@ export const defaultStoreOrdersFields = [
   "total",
 ] as (keyof Order)[]
 
+export const allowedStoreOrdersFields = [
```

Comment:
@fPolic Can I get you to add `billing_address` as an allowed relation? It does not have to be a default one.

Would solve issue [#3182](https://github.com/medusajs/medusa/issues/3182#issuecomment-1418891493)

Reply:
done :)